### ISSUE TITLE
Exited from signal: SIGTERM

### ISSUE THREAD

**Issue by KyleMit** [meteor/windows-preview#29]
I have a clean install of [0.0.27](http://warehouse.meteor.com/windows/install-meteor-0.0.27.exe) on Windows 8.1

When saving a file, the CMD terminal will take a very long time to refresh (if at all) and gives the following output:

```
=> Client modified -- refreshing
=> Exited from signal: SIGTERM
```

---

For example, in the `meteor create --examples todos` project, every time I save, I get the message

>TRYING TO CONNECT
There seems to be a connection issue

**Comment by ignl** [comment]
I think it's duplicate of https://github.com/meteor/windows-preview/issues/7. However I also get "Exited from signal: SIGTERM" message, so exactly same problem for me.

**Comment by avalanche1** [comment]
I've started playing with meteor for win and got the same issue.
However, when reverting to the older win release from win.meteor.com i get the following: "meteor server restarted" when saving changes to files and it takes more than 10secs to refresh the page.
While on mac terminal returns "client modified - refreshing"  and updates browser in around 1sec.

**Comment by stubailo** [comment]
This error message doesn't mean your app is broken, but is related to the problem of very long build times on Windows. We are working on improving the performance soon.

**Comment by avalanche1** [comment]
0.1.1 still restarts the server upon every file change instead of refreshing

**Comment by avital** [comment]
As far as we know, this should be resolved with Meteor 1.1 for Windows, so I'll close this issue.

If you have an issue, please open a new ticket on the [main Meteor repository](https://github.com/meteor/meteor). This repository will be closed soon.
